back/engines/commercial/analytics/README.md

```diff
@@ -0,0 +1,76 @@
+# Analytics
```

Comment:
I will run through this readme to check it's all still correct

Reply:
i think most of it its outdated now since migrations are in the top level directory already, the config file is being updated with the module on the ee repo, and the profile in the docker compose its not being used (maybe we should remove the analytics database from the docker compose file?)